src/credentials/FabricTable.cpp

```diff
@@ -648,9 +649,14 @@ void FabricTable::DeleteAllFabrics()
     }
 }
 
-CHIP_ERROR FabricTable::Init(FabricStorage * storage)
+CHIP_ERROR FabricTable::Init(FabricStorage * storage, FabricTableDelegate * fabricTableDelegate)
```

Comment:
This new arg is used in just one callsite, right?  Why not just have that callsite call the existing AddFabricDelegate() API?

Reply:
It was also added to track down where the fabric table were initialized by triggering an explicit compilation error. But yes, I'll remove it now an add the AddFabricDelegate API